### ISSUE TITLE
Will this selector grab other bots, and not only AutoMod?

### ISSUE THREAD

**Issue by XaeroDegreaz** [purplefloyd14/Not-oModerator#1]
https://github.com/purplefloyd14/Not-oModerator/blob/a0a7e07a34e0935c3638b57ea9f6e37c6a6a43e7/contentScript.js#L4

Such as

![Screenshot_20230509-013328](https://user-images.githubusercontent.com/312459/236879830-93466fdd-7823-4539-9fb0-0695835ba53f.png)

**Comment by purplefloyd14** [comment]
Yes, it will get any post that is made by the mod whose title contains "bot" or "mod". Give it a try and let me know if anything survives that shouldn't!

**Comment by purplefloyd14** [comment]
fixed it for that case! you'll have to redownload and re-add to chrome but it works for that one now
